### PR TITLE
feat: numbering every word in a file

### PR DESCRIPTION
**Pull request by sharma-shruti**

### CODE REVIEW

File: src/numbering_words_in_a_file.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+# Number words across all lines
+
+*I spent 2 pomodoro on this task*
```
Comment (abhi18av) [inline src/numbering_words_in_a_file.md:4]:
```suggestion
Pomodoros: 2
```

**Review by abhi18av — CHANGES_REQUESTED** [review]
Also, maybe it's better to include the contents of the sample file like so

```
file contents 
.
.
.
```

Notes
```
perl stuff
```
Notes
```
powershell stuff
```

**Comment by abhi18av** [comment]
@sharma-shruti , Github automatically closed this PR since I've reworked the `develop` branch git history. 

Now you need to fix this situation and then reopen/re-initiate this PR.

Here are some hints
- `rebase`
- `cherry-pick`
- `patch`

**Comment by sharma-shruti** [comment]
Pomodoro: 3
I think I did something wrong. The file was present in my feature branch so I cherry-picked it on develop branch. But I still don't see this file on origin/develop.
```
Ashwarys-MBP:powershell-one-liners ashwarybhatt$ git cherry-pick d4b8415d47385471c46075855fb4b41e4f6ccd44
[develop d2a05db] feat: numbering every word in a file
 Date: Tue May 5 19:07:20 2020 +0800
 1 file changed, 36 insertions(+)
 create mode 100644 src/numbering_words_in_a_file.md
Ashwarys-MBP:powershell-one-liners ashwarybhatt$ git log
commit d2a05dbef38aa9e05f774ffdc7671fdee79e8542 (HEAD -> develop)
Author: Shruti <shruti26.inbox@gmail.com>
Date:   Tue May 5 19:07:20 2020 +0800

    feat: numbering every word in a file

commit 662a8fae0ee72e6d78422c7b8978e5f263187fa4
Author: Shruti <shruti26.inbox@gmail.com>
Date:   Mon May 4 23:46:17 2020 +0800

    feat: remove blank lines from a file
=============================================
Ashwarys-MBP:powershell-one-liners ashwarybhatt$ git status
On branch develop
Your branch and 'origin/develop' have diverged,
and have 6 and 5 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)

nothing to commit, working tree clean
Ashwarys-MBP:powershell-one-liners ashwarybhatt$
```
When I tried **git pull** it says `no merge base found.`

**Comment by abhi18av** [comment]
@sharma-shruti , you shouldn't cherry-pick onto the `develop` or `master`. Think of both are immutable/unmodifiable branches.

Give this one more pomodoro, then simply 
- delete your local develop / master and pull both again to local
- start new feature branch from the updated develop
- cherry pick only your commits from this branch onto the new feature branch
- initiate another PR

**Comment by sharma-shruti** [comment]
I followed the steps but still unable to push my changes to remote and create a pull request because of the following error.

**Comment by abhi18av** [comment]
@sharma-shruti , no worries. We'll connect tonight for this hmm 

Please continue with the other commands/features

**Pull request closed without merging**